app/views/users/logix/_dashboard.html.erb

```diff
@@ -3,7 +3,7 @@
   <% if policy(project).check_direct_view_access? %>
     <div class="col-xs-12 col-sm-6 col-md-4 col-lg-4">
       <div class="card text-center card-userprojects">
-        <div class="card-header card-header-userprojects">
+        <div class="card-header card-header-userprojects" id="roundoutline">
```

Comment:
In this case there will be multiple components with same id, right?

Reply:
Yes @tachyons . Should I change it ?
![Capture](https://user-images.githubusercontent.com/42182955/76196031-81cdc880-620f-11ea-8e69-e32e107ff637.PNG)

Reply:
Yes

Reply:
@tachyons I will be adding an inline css to the div in order to achieve the border-radius : 0 , because whenever I add a new class or add custom changes to the bootstrap class, the changes wont show up. Please let me know whether it is fine or should I look for another way?